### PR TITLE
Fix wording variant doc

### PR DESCRIPTION
**Pull request by Triplelexx**

### CODE REVIEW

File: development/cpp/variant_class.rst [hunk 1]
```diff
@@ -18,10 +18,10 @@ A Variant can:
 -  Store almost any datatype
 -  Perform operations between many variants (GDScript uses Variant as
    its atomic/native datatype).
--  Be hashed, so it can be compared quickly to over variants
+-  Be hashed, so it can be compared quickly to other variants
 -  Be used to convert safely between datatypes
 -  Be used to abstract calling methods and their arguments (Godot
-   exports all its functions through variants)
```
Comment (NathanLovato) [inline development/cpp/variant_class.rst]: This is the correct form

Comment (mhilbrunner) [reply development/cpp/variant_class.rst]: The formatting also looks a little off there :) And the commits should probably be squashed.

**Comment by mhilbrunner** [comment]
Thanks for your first contribution! Can you please squash your multiple commits into one?
If you don't know how, see http://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html.

If in doubt, feel free to ask for help here or on IRC/Discord!

**Comment by Triplelexx** [comment]
@mhilbrunner Oh ok, not something I've done before, I thought the maintainer was responsible for that.
I'll have a look.

**Comment by Triplelexx** [comment]
Good info in that link. Thanks 👍

**Comment by NathanLovato** [comment]
Don't worry for this, squash and merge should be okay on the docs - it's less strict than the engine's repo

**Pull request merged**